Libraries/Opc.Ua.Gds.Server.Common/CertificateGroup.cs

```diff
@@ -175,25 +175,25 @@ public virtual async Task Init()
         public async virtual Task<X509CRL> RevokeCertificateAsync(
             X509Certificate2 certificate)
         {
-            Task<X509CRL> crl = RevokeCertificateAsync(
+            X509CRL crl = await RevokeCertificateAsync(
                 AuthoritiesStorePath,
                 certificate,
-                null);
+                null).ConfigureAwait(false);
 
-            //Also update TrustedList CRL so registerd Applications can get the new CRL
-            await crl.ContinueWith((_) =>
-                UpdateAuthorityCertInCertificateStore(Configuration.TrustedListPath)
-                , TaskContinuationOptions.OnlyOnRanToCompletion).ConfigureAwait(false);
-
-            //Also update TrustedIssuerCertificates Store
-            if (!String.IsNullOrEmpty(TrustedIssuerCertificatesStorePath))
+            // Also update TrustedList CRL so registerd Applications can get the new CRL
+            if (crl != null)
             {
-                await crl.ContinueWith((_) =>
-                    UpdateAuthorityCertInCertificateStore(TrustedIssuerCertificatesStorePath)
-                    , TaskContinuationOptions.OnlyOnRanToCompletion).ConfigureAwait(false);
+                await UpdateAuthorityCertInCertificateStore(Configuration.TrustedListPath).ConfigureAwait(false);
+
+                //Also update TrustedIssuerCertificates Store
+                if (!String.IsNullOrEmpty(TrustedIssuerCertificatesStorePath))
+                {
+                    await UpdateAuthorityCertInCertificateStore(TrustedIssuerCertificatesStorePath).ConfigureAwait(false);
```

Comment:
since we are in 'breaking change' mode, the Async suffix could be added to the function to make the Asnyc more obvious.

Reply:
maybe we do Async suffixes this in a seperate PR